Grupo 5 - Scripts

pinguim11 opened the pull request:
adição das constraints da tabela material

pinguim11 committed:
Update 02_constraints_material.sql

adição das constraints da tabela material

Pietro-MM commented on esquema_fisico/02_constraints_material.sql:
- A tabela material existe mas ainda não foi declarada, recomendo a produação da mesma. Adicionar junto nesta PR.

pinguim11 committed:
Update 02_constraints_material.sql

ajuste dos nomes das variáveis material_anterior para material_anterior_id e codigo_componente para componente_curricular_codigo

pedro-salviano reviewed (approved):
LGTM!
